.github/workflows/ci.yml

```diff
@@ -27,10 +27,15 @@ jobs:
           cache: yarn
 
       - run: yarn install --frozen-lockfile
-      - name: Lint
+
+      - name: Lint Addon
         run: yarn lint:js
         working-directory: packages/ember-cookies
 
+      - name: Lint Test App
+        run: yarn lint:js
+        working-directory: packages/test-app
+
```

Comment:
I think the linting of the addon and app should run in parallel – currently I'm not sure the lint job for the app would run at all if the one for the addon fails?

Reply:
@marcoow ok, I can do it: would you like to introduce two new lint jobs: for addon and test app *or* only extract addon lint into separate job so test app lint + tests grouped?

Reply:
@marcoow this is solved - the linting of the addon and app should run in parallel and is setup via `lint` script in the root `package.json`

Reply:
It still seems like we're running at least the linting sequentually? https://github.com/simplabs/ember-cookies/runs/6499932915?check_suite_focus=true

Reply:
Yeah, I'd split into separate jobs 👍

Reply:
@marcoow did split into `lint-addon` and `lint-test-app` jobs that run in parallel on separate agents. Does this look good to you?